### ISSUE TITLE
(emergency) Part of parameters are not saved

### ISSUE THREAD

**Issue by qiminchen** [xiumingzhang/GenRe-ShapeHD#39]
Hi, I defined some the Net class like the following, after training, I found out that the parameters of some networks I defined, for example `self.fc` and `self.classifier`,  in Net class are not saved because when I resume training, the loss and accuracy didn't start around where it stopped, do you have any suggestions where to look at?

```
class Net(nn.Module):
    def __init__(self, npf, npc, ndims):
        super().__init__()
        self.Encoder = VoxelEncoder(ndims=ndims).cuda()
        self.classifier = {}
        self.fc = FC(in_dims=ndims, out_dims=100).cuda()
        for i in range(2):
            self.classifier[i] = ClassificationLayer(100, npc[i]).cuda()
```

**Comment by ztzhang** [comment]
Hi, 
I'm not sure how you wrote your stuff, but by default the code would save networks defined in self._net. I guess you defined a network but did not register it in self._net list.
